lib/main.js

```diff
@@ -532,15 +523,15 @@ function GetMML(result) {
 }
 
 //
-//  Creates the speech string and updates the MathML to include it, if needed
+//  Creates dummy speech string and updates the MathML to include it, if needed
 //
 function GetSpeech(result) {
-  if (!data.speakText) return;
-  speech.setupEngine({semantics: true, domain: data.speakRuleset, style: data.speakStyle});
-  result.speakText = speech.toSpeech(result.mml);
+  result.speakText = "Equation";
+  if (data.format !== "MathML") result.speakText = data.math;
```

Comment:
Note that it is possible for this to make the `speakText` be blank (if `data.math` is blank, e.g., from `\(\)` and input is TeX).